src/main/java/org/jabref/gui/preftabs/AppearancePrefsTab.java

```diff
@@ -247,8 +253,12 @@ public void setValues() {
     public void storeSettings() {
         // L&F
         prefs.putBoolean(JabRefPreferences.USE_DEFAULT_LOOK_AND_FEEL, !customLAF.isSelected());
+        final boolean oldFxTweakValue = prefs.getBoolean(JabRefPreferences.FX_FONT_RENDERING_TWEAK);
+        prefs.putBoolean(JabRefPreferences.FX_FONT_RENDERING_TWEAK, fxFontTweaksLAF.isSelected());
         prefs.put(JabRefPreferences.WIN_LOOK_AND_FEEL, classNamesLAF.getSelectedItem().toString());
-        if ((customLAF.isSelected() == useDefaultLAF) || !currentLAF.equals(classNamesLAF.getSelectedItem().toString())) {
+        if ((customLAF.isSelected() == useDefaultLAF)
```

Comment:
Can you extract this into a private method named something like `isRestartRequired`?